fossunited/foss_hackathon/doctype/foss_hackathon/test_foss_hackathon.py

```diff
@@ -0,0 +1,23 @@
+import frappe
+from frappe.tests import IntegrationTestCase
+
+from fossunited.doctype_ids import HACKATHON
+from fossunited.tests.utils import insert_test_hackathon
+
+
+class TestFOSSHackathon(IntegrationTestCase):
+    def setUp(self):
+        self.hackathon = insert_test_hackathon()
+
+    def tearDown(self):
+        frappe.set_user("Administrator")
+        frappe.delete_doc(HACKATHON, self.hackathon.name, force=True)
+
+    def test_hackathon_route(self):
+        if self.hackathon.permalink:
+            self.assertTrue(self.hackathon.route == f"hack/{self.hackathon.permalink}")
```

Comment:
```suggestion
            self.assertEqual(self.hackathon.route, f"hack/{self.hackathon.permalink}")
```

same comment with the other `assertEqual` in this testcase.